protos/dcs/hook/v0/hook.proto

```diff
@@ -87,6 +102,30 @@ message SetPausedRequest {
 message SetPausedResponse {
 }
 
+message ReloadCurrentMissionRequest {
+}
+
+message ReloadCurrentMissionResponse {
+}
+
+message LoadNextMissionRequest {
+}
+
+message LoadNextMissionResponse {
+  // Was the next mission successfully loaded. SHOULD return false when the
+  // end of the mission list has been reached but DCS appears to always
+  // return true
+  bool loaded = 1;
+}
+
+message LoadMissionRequest {
+  // The full path to the .miz file to be loaded
+  string file_name = 1;
```

Comment:
Shall we rename this to `file_path` as this is the full path to where the `.miz` is located?

Reply:
Yeeaaah, I was never convinced about this naming. I stuck with it since that is what ED uses and I was wondering if we would want to limit the path to the Mission dir or something to avoid potential issues with loading files it shouldn't be.

Reply:
Deferred to https://github.com/DCS-gRPC/rust-server/issues/154 . For `0.6.0` we will use the above as-is.